misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx

```diff
@@ -26,12 +26,35 @@ const RawMaterialIssueActivity = (props) => {
   const showHazardsAndAllergens =
     lines && lines.some((lineItem) => lineItem?.hazardSymbols?.length > 0 || lineItem?.allergens?.length > 0);
 
+  const getDisabledStatus = useCallback(({ currentLine, previousLine }) => {
+    if (isAlwaysAvailableToUser) {
+      return false;
+    }
+    if (!isUserEditable) {
+      return true;
+    }
+    if (currentLine.completeStatus === CompleteStatus.COMPLETED) {
+      return false;
+    }
+    return previousLine && previousLine.completeStatus !== CompleteStatus.COMPLETED;
+  }, []);
```

Comment:
pls include isAlwaysAvailableToUser, isUserEditable to dependencies array.... or don't use useCallback

Reply:
ahh, i've read again the code... basically there is no point to for useCallback here.

Reply:
yes, you are right, it was a left over from a previous commit, thx for pointing it out